src/AdminConsole/Pages/Organization/Overview.cshtml

```diff
@@ -1,10 +1,13 @@
 @page
+@using Passwordless.AdminConsole.Components
 @model OverviewModel
 
 @{
     ViewData["Title"] = Model.Org.Name + " Applications";
 }
 
+<component type="typeof(ReleaseNotes)" render-mode="ServerPrerendered"/>
+

```

Comment:
This is OK but not the prettiest. In Blazor components you just do `<ReleaseNotes />`

Reply:
@abergs can you explain what that line does? Does it specify that the component should be prerendered, as opposed to everything else?

Reply:
@Tyrrrz Supposedly it first renders the component statically, then refreshes the component again to make it interactive.